src/Comment.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import './Comment.css';
+
+class Comment extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: props.commentId,
+      by: null,
+      text: null,
+      time: null,
+      error: null
+    };
+
+    this.componentDidMount = this.componentDidMount.bind(this);
+    this.getData = this.getData.bind(this);
+  }
+
+  getData() {
+    fetch(`https://hacker-news.firebaseio.com/v0/item/${this.state.id}.json`)
+      .then(res => res.json())
+      .then(
+        (resJson) => {
+          if (resJson.text) {
+            let text = resJson.text;
```

Comment:
Anything that can be a const should be in es6 land

Reply:
This no longer needs to be a variable to begin with.